Update access-management.md (#1764)
RedisLabs/redislabs-docs: content/rc/administration/access-management.md

To grant Redis Cloud access to a GCP user, select the **Add** button to add a member, insert the email address, and then assign the following roles to the user:

> - An 'Owner' or 'Editor' role will allow the user to access the Redis console and perform changes.

Reviewer: ```suggestion
- 'Owner' and 'Editor' roles let users access the Redis admin console and make changes.
```

> - A 'Viewer' role will allow the user to access the Redis console but not to make any changes.

Reviewer: ```suggestion
- A 'Viewer' role lets the user access the Redis admin console but not make changes.
```

> - A user with a 'Browser' role will not be able to access the Redis console.

Reviewer: ```suggestion
- A user with a 'Browser' role cannot access the Redis admin console.
```
